awx/ui_next/src/contexts/Session.jsx

```diff
@@ -73,15 +73,12 @@ function SessionProvider({ children }) {
   const [authRedirectTo, setAuthRedirectTo] = useState('/');
 
   const logout = useCallback(async () => {
-    if (!isSessionExpired.current) {
-      history.replace('/');
-    }
```

Comment:
talked w/ @marshmalien 

This code makes sure you land on the expected page when signing back in.